README.md

```diff
@@ -486,6 +486,19 @@ ChatView(
 ```
 
 
+20. Field `cancelRecordIcon` to provide an icon for cancel voice recording.
+
+```dart
+ChatView(
+   ...
+      voiceRecordingConfiguration: const VoiceRecordingConfiguration(
+        cancelRecordIcon: const Icon(Icons.cancel),
```

Comment:
Please create cancelRecordConfig class and add icon and onCancel parameters to it.